src/redux/ConfigureStore.js

```diff
@@ -0,0 +1,13 @@
+import { configureStore } from '@reduxjs/toolkit';
+import booksReducer from './books/books';
+// eslint-disable-next-line
+import categoriesReducer from "./categories/categories";
+
```

Comment:
- [ ] The way you are importing the `categoriesReducer` is incorrect. The correct path should be:

`"./Categories/categories"`. In your path for the import statement, you are using lowercase _**c**_ instead of uppercase _**C**_ for the `Categories` directory word. This causes the error I attached in the screenshot below


![image](https://user-images.githubusercontent.com/77106962/217838618-fc955ffa-6c5b-492b-90ef-75b641438ef4.png)

#### Kindly fix this error 👍 😎